### PR TITLE
[TF-XLA:CPU][oneDNN] Disable grappler remapper fusions on TF->XLA CPU JIT compilation

### PR DESCRIPTION
**Pull request by mdfaijul**
This PR disables remapper fusions on CPU, when TF model is set for XLA JIT compilation. TF fused ops cannot be translated into XLA instructions. The fusions are carried out by XLA CPU compiler passes.

### CODE REVIEW

File: tensorflow/core/grappler/optimizers/remapper_test.cc [hunk 1]
```diff
@@ -3044,5 +3044,69 @@ TEST_F(RemapperControlDependencyPatternMatcher, BF16) {
...
+    TF_ASSERT_OK(s.ToGraphDef(&item.graph));
+
+    const string device = "/device:CPU:0";
+
+    // Place all nodes on CPU.
+    for (int i = 0; i < item.graph.node_size(); ++i) {
+      item.graph.mutable_node(i)->set_device(device);
+    }
+
+    Remapper optimizer(RewriterConfig::ON, RewriterConfig::NO_CONVERSION_ON_CPU,
```
Comment (cantonios) [inline tensorflow/core/grappler/optimizers/remapper_test.cc:3087]: nit: the convention for parameter comments is to use `/*parameter_name=*/value` (e.g. `/*xla_auto_clustering_on=*/true`.  This allows our linter tools to catch mismatched parameter names.

File: tensorflow/core/grappler/optimizers/remapper_test.cc [hunk 2]
```diff
@@ -3044,5 +3044,69 @@ TEST_F(RemapperControlDependencyPatternMatcher, BF16) {
   RunTest<DT_BFLOAT16>();
 }
 
+class XlaCpuJitDisableFusionTest : public RemapperTest {
+ protected:
+  void SetUp() override {
+    setenv("TF_XLA_FLAGS", "--tf_xla_cpu_global_jit", 1 /* replace */);
```
Comment (cantonios) [inline tensorflow/core/grappler/optimizers/remapper_test.cc:3050]: nit: `/*overwrite=*/1`

(see below)

**Review by cantonios — CHANGES_REQUESTED** [review]

File: tensorflow/core/grappler/optimizers/remapper_test.cc (hunk 2 above)
Comment (mdfaijul) [reply tensorflow/core/grappler/optimizers/remapper_test.cc]: Done.

File: tensorflow/core/grappler/optimizers/remapper_test.cc (hunk 1 above)
Comment (mdfaijul) [reply tensorflow/core/grappler/optimizers/remapper_test.cc:3087]: Done.

File: tensorflow/core/grappler/optimizers/remapper.cc [hunk 3]
```diff
@@ -4793,15 +4821,29 @@ bool RequiresInferredShapes(const RemapperContext& ctx, int node_index,
...
+  return std::find(tf_xla_flags.begin(), tf_xla_flags.end(),
+                   tf_xla_cpu_global_jit) != tf_xla_flags.end();
+}
 }  // namespace
 
 Status Remapper::Optimize(Cluster* cluster, const GrapplerItem& item,
                           GraphDef* optimized_graph) {
   GrapplerItem mutable_item = item;
   Status status;
+  bool xla_cpu_jit_disble_fusion =
```
[the remark this answers is not in the document]
Comment (mdfaijul) [reply tensorflow/core/grappler/optimizers/remapper.cc]: Done.

**Comment by mdfaijul** [comment]
@cantonios Thanks for the review. I have address the comments.

**Review by cantonios — APPROVED** [review]

**Pull request merged**